src/main/java/org/fcrepo/camel/processor/SparqlUpdateProcessor.java

```diff
@@ -58,6 +58,7 @@ public void process(final Exchange exchange) throws IOException {
         serializer.serialize(serializedGraph, graph.getGraph(), "text/rdf+nt");
 
         exchange.getIn().setBody("DELETE WHERE { <" + subject + "> ?p ?o };\n" +
+                                 "DELETE WHERE { <" + subject + "/fcr:export?format=jcr/xml> ?p ?o };\n" +
```

Comment:
This looks like a (valid) hack for plucking out specific triples. If so, please add a comment to the effect of why you are requiring the fcr:export path element. There is a good chance that the jcr/xml export may be going away at some point, and it is important that we know if this line of code is a true dependency.